Pocket/pocket-ios
# fix(navigation within webview): change reader actions when navigating within webview [IN-997]

## Summary
Modify reader actions when navigate to different articles within Webview.

## References 
[IN-997](https://getpocket.atlassian.net/browse/IN-997)

## Implementation Details
We are fetching an item based on URL from coredata and adjust share sheet options accordingly.

## Test Steps
when I view Home and it shows recommendations 
AND I should be able to click on Recommendation, that opens in Webview and clicking on 'Share' options should show a list of Activities and list should have custom activities - 'Save' & 'Report'
then If I navigate to another article, I should not be seeing any reader activities withing webview (ex: save, report etc)

## PR Checklist:

## Screenshots
| Recommendation (When open)        | After navigate to another article |
|-------------|-------------|
| ![Simulator Screen Shot - iPhone 14 - 2022-11-14 at 08 07 36](https://user-images.githubusercontent.com/110417980/201668506-d12f8980-ab50-4f1a-870e-c9105c21a9a1.png) | ![Simulator Screen Shot - iPhone 14 - 2022-11-14 at 08 11 53](https://user-images.githubusercontent.com/110417980/201668910-57e90157-ec61-4bbf-b525-c71e77c5b7f1.png) |
